app/controllers/workouts_controller.rb

```diff
@@ -1,7 +1,7 @@
 class WorkoutsController < ApplicationController
   def create
     # need to add a check for sets. Is it an appropriate number?
-    blueprint = Blueprint.find(params[:id])
+    blueprint = Blueprint.find(params[:blueprint_id])
```

Comment:
Could you also update the frontend code in this PR to pass this field as well? Just want to make it so things don't break.